### PR TITLE
Run without appcache

### PR DESCRIPTION
**Pull request by DileSoft**
Saves now working!

You can play the game on http://candybox2.dilesoft.ru

### CODE REVIEW

**Comment by DileSoft** [comment]
Really it is bit hacked. In fact I just remove application cache check in https://github.com/candybox2/candybox2.github.io/blob/a3031f72a6575659066e729f16dd462d07fc9609/code/main/LocalSaving.ts#L72 and in candybox2.js which I copied and pasted from http://candybox2.github.io/

File: code/main/LocalSaving.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ module LocalSaving{
     }
     
     export function supportsLocalSaving(): boolean{
-        if('localStorage' in window && window['localStorage'] !== null && 'applicationCache' in window && window['applicationCache'] !== null)
```
Comment (kvaps) [inline code/main/LocalSaving.ts:72]: So this change is enough to make save function working again?🤔

I can cherry-pick only this, if you want to

**Review by kvaps — COMMENTED** [review]
HI @DileSoft, thank you for your PR!

Could you please rebase, to exclude compiled files and separate the changes for fixing the save functionality and improving build process into separate PR.

I will update the version and recompile it in separate commit.

File: code/main/LocalSaving.ts (hunk 1 above)
Comment (DileSoft) [reply code/main/LocalSaving.ts:72]: Yes, it's enough. Please cherry-pick it.

**Comment by Elirector** [comment]
> Saves now working!
> 
> You can play the game on http://candybox2.dilesoft.ru

please, fix load links, they lead to http://candybox2.github.io

**Comment by kvaps** [comment]
merged into `v1.2.3`, thank you!

**Pull request merged**